meta-lmp-base/recipes-bsp/trusted-firmware-a/tf-a-tools-fio_2.6.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "Cert_create & Fiptool for fip generation for Trusted Firmware-A"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://license.rst;md5=1dd070c98a281d18d9eefd938729b031"
+
+SRC_URI = "git://github.com/ARM-software/arm-trusted-firmware.git;protocol=https;nobranch=1"
+#SRCREV corresponds to v2.6
```

Comment:
tf-a 2.7 and tools 2.6?

Reply:
Yes, tools 2.6 fail to build (investigating now)

Reply:
I am on it as well because it fails in main-next too